### PR TITLE
Add docs for defining upper-case variables on nested Settings models

### PR DESCRIPTION
**Pull request by aminalaee**
## Change Summary

The reason explained here: https://github.com/samuelcolvin/pydantic/issues/3936#issuecomment-1152903692

## Related issue number

Add docs for #3936

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [ ] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

**Comment by aminalaee** [comment]
please review

**Review by hramezani — APPROVED** [review]

File: docs/usage/settings.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ so in this example
 all upper-case, you should name attribute all upper-case too. You can still name environment variables anything
 you like through `Field(..., env=...)`.
 
+In Pydantic **v1** `case_sensitive` is `False` by default and all variable names are converted to lower-case internally.
+If you want to define upper-case variable names on nested models like `SubModel` you have to
+set `case_sensitive=True` to disable this behaviour.
```
Comment (hramezani) [inline docs/usage/settings.md:61]:
```suggestion
set `case_sensitive=True` to disable this behavior.
```

**Review by hramezani — CHANGES_REQUESTED** [review]
Thanks @aminalaee for the PR :+1: 
Please update your PR and fix the small typo.

**Comment by hramezani** [comment]
please update

**Review by hramezani — APPROVED** [review]

**Comment by aminalaee** [comment]
Thanks @hramezani 🚀

**Pull request merged**